bin/stringfuzzx

```diff
@@ -122,7 +123,14 @@ def main():
     )
     # graft transformer
     graft_parser = subparsers.add_parser(GRAFT, help='graft transformer')
-
+    graft_parser.add_argument(
+        '--str-to-re',
+        dest    = 'skip_str_to_re',
+        action  = 'store_false',
+        default = DEFAULT_SKIP_STR_TO_RE,
+        help    = 'Include str_to_re nodes in grafting (default: {})'.format(DEFAULT_SKIP_STR_TO_RE)
```

Comment:
The description doesn't match the name. Do you specify the flag to skip, or to include?

Reply:
Yeah would you be OK if I just got rid of all these flags?

Reply:
Instead I just fixed it. Merge if you accept. I'll get started on the merge transformer now.